src/styles/base/_settings.scss

```diff
@@ -10,3 +11,4 @@ $large-space: 100px;
 //Demo spacing in percentage
 $large-ratio: 100%;
 $Xlarge-ratio: 300%;
+$font-Family: 'Reem Kufi', sans-serif;
```

Comment:
Give this font-family a more descriptive and specific name. There can be more than one font-family in our project.
`$reem-kufi: 'Reem Kufi', sans-serif;`

Also add new line to end of file.